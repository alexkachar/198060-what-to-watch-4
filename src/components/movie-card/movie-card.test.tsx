import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';
import MovieCard from './movie-card';
import MOCK_MOVIES from '../../test-data/mock-movies';

it(`MovieCard renders correctly`, () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <MovieCard movie={MOCK_MOVIES[0]} />
      </BrowserRouter>, {
      createNodeMock: () => document.createElement(`div`)
    })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
