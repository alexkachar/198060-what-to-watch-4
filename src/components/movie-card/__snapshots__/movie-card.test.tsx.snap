// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <a
    href="/film/1"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="The Grand Budapest Hotel"
        height={175}
        src="img/the-grand-budapest-hotel.jpg"
        width={280}
      />
    </div>
  </a>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/film/1"
      onClick={[Function]}
      style={
        Object {
          "color": "#c9b37e",
        }
      }
    >
      The Grand Budapest Hotel
    </a>
  </h3>
</article>
`;
